docs/variants/novacustom_nv4x/building.md

```diff
@@ -21,48 +21,36 @@ Obtain the image:
 docker pull coreboot/coreboot-sdk:0ad5fbd48d
 ```
 
-Obtain coreboot source code for NovaCustom NV4X:
+Clone the coreboot repository:
 
 ```bash
-git clone https://github.com/Dasharo/coreboot.git
+git remote add dasharo https://github.com/dasharo/coreboot.git
```

Comment:
@mkopec The clone is entirely removed. Intentional?

Reply:
no, sorry. fixed